src/builders/browser/index.ts

```diff
@@ -40,13 +41,15 @@ export class SingleSpaBrowserBuilder extends BrowserBuilder {
       config.plugins.splice(indexHtmlWebpackPluginIndex, 1);
     }
 
-    // Configure Wepack output
-    config.output = {
-      library: options.libraryName,
-      libraryTarget: options.libraryTarget
-    };
-
-    return config;
+    return webpackMerge.smart(config, {
```

Comment:
See https://github.com/survivejs/webpack-merge#smart-merging.

The previous code overwrote the entire output object, but now we just merge in the library and libraryTarget. See https://webpack.js.org/configuration/output/#root for more details

Reply:
Cool! 👍